dedibox-network/rpn/how-to/find-rpn-gateway.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: How to find your RPN gateway
+description: This page explains how to find your RPN gateway
+dates:
+  validation: 2021-11-10
+  posted: 2021-11-10
+---
+
+Your RPNv1 gateway may differ from one Dedibox to another for techical reasons.
+It is normally pushed by DHCP, and and configured automatically.
+But in some cases you may need to know it.
+
+<Message type="requirement">
+
+* You have an account and are logged into the [Dedibox Console](https://console.online.net)
+* You have a Dedibox dedicated server with RPN interface
+
+</Message>
+
+You can find the RPNv1 gateway for your Dedibox in [rescue mode](/dedibox/dedicated-servers/how-to/use-rescue-mode/) by typing:
+
+```
+# ip route show
+10.90.0.0/XX via X.X.X.X ...
+```
+
+If you do not know your RPN netmask, you can find it by using IPCALC. The gateway is always the first IP of the subnet.
+```
+# ipcalc -b [SET_HERE_YOUR_RPN_IP]/[NETMASK CIDR] | grep HostMin
+HostMin: X.X.X.X
+```
+The gateway is in place of `X.X.X.X`.
+
+<Message type="tip">
+
+The gateway does not ping, this is normal behavior.
+
+</Message>
```

Comment:
It's a bit unclear for me what is part of each command, what is the output and what bits need to be replaced with users own values. Maybe something like:

1. Enter rescue mode (link to how)

2. Enter the following command to display your RPNv1 gateway (detailing any substitutions to make):

```
command
```

Your output should be similar to the following and shows you bla bla

```
output
```

3. Enter the following command to display your RPN netmask (detailing any substitutions to make):

```
ipcalc bla bla bla
```

Your output should be similar to the following and shows you bla bla

```
output
```

I don't know if the "gateway does not ping" message is a consequence of the ipcalc command or the ip route show command?

Do they need the netmask in order to carry out the first command for the RPNv1 gateway?

Reply:
I will update this. After proofreading it does not make much sensse